### DISCUSSION TITLE
Slow file download speed.

### DISCUSSION THREAD

**Discussion by TXNNGVGX** [community/community#78368] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello. Recently, the download speed of any files from any repositories has dropped significantly. My provider links to github servers. How do I find out? _188.243.71.180_

**Comment by HiWebDev0026** [comment, marked as answer]
Hi there!
To troubleshoot and pinpoint the cause of this issue, you can consider the following steps:

Use speed test tools: Begin by using online speed test tools to measure the current download and upload speeds of your internet connection. This will provide you with an overview of your network's performance, allowing you to compare it with the expected speeds from your internet service provider.

Traceroute to GitHub servers: Use the traceroute command in your computer's terminal or command prompt to trace the network path to the GitHub servers. This will help you identify any network hops or delays along the way, allowing you to pinpoint where the slowdown might be occurring.

Check for provider issues: Reach out to your internet service provider and inquire about any potential network issues or maintenance activities that could be affecting the performance of connections to GitHub servers and other repositories.

Utilize alternate connections: Try downloading files from other repositories or using a different internet connection to see if the slowdown is specific to the link between your provider and GitHub servers. This can help differentiate whether the issue is related to your local network or the provider's network.

Monitor for consistency: Keep track of the download speeds over a period of time to see if the slowdown persists or fluctuates. This information can be helpful in diagnosing the issue and providing your provider with detailed data.

By employing these steps, you can gather valuable information to help identify the cause of the slowdown and work towards resolving the issue with your provider if necessary.

**Reply by sybrew** [reply]
Provided a GPT answer to prove "legitimacy" on their profile. They also cloned multiple repos from other developers without proper attribution. HiWebDev0026's account is setup only for phishing.

**Reply by omergoktas** [reply]
Reported the b!tch -sick of bots.
